rockcraft.yaml

```
@@ -0,0 +1,68 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+---
+name: charmed-kafka-ui
+base: ubuntu@24.04
+version: '1.3.0'
+summary: Charmed Kafka Rock OCI
+description: |
+  This is an OCI that provides `kafbat/kafka-ui`, intended for use in the
+  Charmed Kafka UI K8s Charmed Operator.
+license: Apache-2.0
+
+platforms:
+  amd64:
+
+services:
+  kafka-ui:
+    override: replace
+    command: |
+      java \
+        -Dspring.config.additional-location=$KAFKA_UI_CONFIG \
+        --add-opens java.rmi/javax.rmi.ssl=ALL-UNNAMED \
+        $JAVA_OPTS \
+        -jar /opt/kafka-ui/libs/api-1.3.0.jar
+    startup: enabled
+    user: kafka-ui
+    group: kafka-ui
+    environment:
+      JAVA_HOME: /usr/lib/jvm/java-21-openjdk-amd64
+      KAFKA_UI_CONFIG: /etc/kafka-ui/application-local.yml
+
+parts:
+  kafka-ui:
+    plugin: nil
+    stage-packages:
+      - openjdk-21-jdk
+      - libpsl5
+      - util-linux
+      - curl
+    build-environment:
+      - JAVA_HOME: /usr/lib/jvm/java-21-openjdk-amd64
+    stage-snaps:
+      - charmed-kafka-ui/latest/edge
+    override-build: |
+      ln -s /usr/lib/jvm/java-21-openjdk-amd64/bin/java \
+        $CRAFT_PART_INSTALL/usr/bin/java
+     ln -s /usr/lib/jvm/java-21-openjdk-amd64/bin/keytool 
```

Sorry I forgot the trailing backslash :(

```suggestion
      ln -s /usr/lib/jvm/java-21-openjdk-amd64/bin/keytool \
```